src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/HealthCertificateValidationCoordinator.swift

```diff
@@ -95,10 +96,12 @@ final class HealthCertificateValidationCoordinator {
 						}
 					case .failure(let error):
 						switch error {
-						case .TECHNICAL_VALIDATION_FAILED:
+						case .TECHNICAL_VALIDATION_FAILED(let expirationDate, let signatureInvalid):
```

Comment:
```suggestion
						case let .TECHNICAL_VALIDATION_FAILED(expirationDate, signatureInvalid):
```
so you don't need the swiftlint disable pattern_matching_keywords